lib/puppet-lint/plugins/check_absolute_classname.rb

```diff
@@ -40,6 +40,29 @@ def check
           end
           s = s.next_token
         end
+      elsif token.type == :INHERITS
+        s = token.next_code_token
+        if s.type == :NAME && s.value.start_with?('::')
+          notify :warning, {
+            message: message,
+            line: s.line,
+            column: s.column,
+            token: s
+          }
+        end
+      elsif token.type == :TYPE and token.value == 'Class' and token.next_code_token.type == :LBRACK
+        s = token.next_code_token
+        while s.type != :RBRACK
+          if s.type == :SSTRING && s.value.start_with?('::')
+            notify :warning, {
+              message: message,
```

Comment:
We should probably use a different message here?
`class included by absolute name` isn't accurate.